### ISSUE TITLE
test gateway as a replacement for tomcat@proxycas

### ISSUE THREAD

**Issue by landryb** [georchestra/ansible#128]
items:
- as both the s-p & the gateway should be available in 24.0 (cf georchestra/improvement-proposals#8) it should be a toggle in the playbook
- gateway/security.yaml needs to be templated for at least ldap baseDn / admin dn/pass
- systemd unit for gateway microservice
- nginx vhost from #127 should point to the gateway port instead of 8180 that was used by proxycas (or make gateway listen on 8180 ?)
- logging to `/srv/log` like other microservices
- what options to put in the env for spring boot startup ?

**Comment by pmauduit** [comment]
cf https://github.com/georchestra/ansible/tree/gateway for the work branch

**Comment by landryb** [comment]
tested locally, needed this chunk for the gw to start:
```
--- a/gateway/routes.yaml
+++ b/gateway/routes.yaml
@@ -37,10 +37,10 @@ spring:
         uri: ${georchestra.gateway.services.geoserver.target}
         predicates:
         - Path=/geoserver/**
-      - id: geowebcache
-        uri: ${georchestra.gateway.services.geowebcache.target}
-        predicates:
-        - Path=/geowebcache/**
+          #- id: geowebcache
+          #uri: ${georchestra.gateway.services.geowebcache.target}
+          #predicates:
+          #- Path=/geowebcache/**
```
some headers are apparently needed for geoserver, otherwise going to https://georchestra.example.org/geoserver redirects to http://localhost:8380/geoserver/web (even with `X-Forwarded-Host`/`X-Forwarded-Proto`)

other than that logging in and loading console/geonetwork/mapstore seems to work. a toggle in the playbook to decide whether we want to use the s-p or the gw (and send traffic to 8180 or 8980 accordingly) is still needed

**Comment by landryb** [comment]
@pmauduit can you do a pr from the gateway branch rebased on master so that we can merge it ? please also grab the gwc bit from georchestra/datadir#395 ;)

**Comment by landryb** [comment]
closed by #134
